latex/chain.tex

```diff
@@ -694,10 +931,13 @@ \subsection{Block Body Transition}
       \var{txs} \leteq \bbody{block}
       &
       \var{vk} \leteq \bissuer{bhb}
+      &
+      \var{hk} \leteq \hashKey{vk}
       \\
-      \bSize{bh} < \fun{maxBBSize}~\var{pp}
```

Comment:
yes,` bBodySize` is correct. I think this method is called `bSize` in the byron spec.